source/whats-new.txt

```diff
@@ -48,11 +48,21 @@ What's New in 6.0
 
 The {+driver-short+} v6.0 release includes the following features:
 
+.. important:: Deprecation Notice
+   
+   All of the ``ssl`` prefixed options in the ``MongoClientOptions``
```

Comment:
Issue: 
I think in order to avoid confusion, a hyphen would be necessary for the modifier since "prefixed" could mean something on its own.


Suggestion:
```suggestion
   All of the ``ssl``-prefixed options in the ``MongoClientOptions``
```

Applies to "tls-prefixed" as well.